scripts/publish.py

```diff
@@ -49,6 +49,7 @@
     "wasmer-c-api": set(["wasmer", "wasmer-compiler", "wasmer-compiler-cranelift", "wasmer-compiler-singlepass",
                          "wasmer-compiler-llvm", "wasmer-emscripten", "wasmer-engine", "wasmer-engine-jit",
                          "wasmer-engine-native", "wasmer-engine-object-file", "wasmer-wasi", "wasmer-types"]),
+    "wasmer-middlewares": set(["wasmer", "wasmer-types", "wasmer-vm"]),
```

Comment:
(No action required:) huh, so beta1 didn't have the middleware? Were the users using the revision from git then?

Reply:
I think yes.

Reply:
Yeah, it seems so!  We updated the version but did not publish it last time